pulp_smash/tests/rpm/api_v2/test_sync_publish.py

```diff
@@ -265,6 +266,9 @@ def setUpClass(cls):
             cls.sync_repo,
         )
         cls.task_bodies = tuple(poll_spawned_tasks(cls.cfg, report))
+        cls.repos_iter = tuple((
+            get_repo(cls.cfg, attrs['_href']) for attrs in cls.attrs_iter
+        ))
```

Comment:
I'm definitely not crazy about the readability here, and I only did it this way to be consistent with ``self.attrs_iter``. I'm very happy to do it differently.

That said, now I really don't understand what ``self.attrs_iter`` is supposed to be. The name is not descriptive, and your example seems to have assigned completely different data to it than before.

Can we just not use that attribute name? It's confusing, and doesn't seem to be helping. What if the last line is:

``self.repo = get(cls.cfg, repo_href)``